setup.py

```diff
@@ -25,6 +25,7 @@
     entry_points={
         'console_scripts': [
             'vivbin=vivisect.vivbin:main',
+            'vivserver=vivisect.remote.server:main',
```

Comment:
If you're going to expose remote servers via an entry script, I'm going to say you need to provide the docs for users on how to start and use it :)

Reply:
lol.  sounds good to me.

Reply:
done